packages/react-components/react-persona/src/components/Persona/usePersonaStyles.ts

```diff
@@ -1,33 +1,192 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
 import type { PersonaSlots, PersonaState } from './Persona.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const personaClassNames: SlotClassNames<PersonaSlots> = {
   root: 'fui-Persona',
-  // TODO: add class names for all slots on PersonaSlots.
-  // Should be of the form `<slotName>: 'fui-Persona__<slotName>`
+  avatar: 'fui-Persona-avatar',
+  presence: 'fui-Persona-presence',
+  primaryText: 'fui-Persona-primaryText',
+  secondaryText: 'fui-Persona-secondaryText',
+  tertiaryText: 'fui-Persona-tertiaryText',
+  quaternaryText: 'fui-Persona-quaternaryText',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'inline-grid',
+    gridAutoColumns: 'max-content',
+    gridAutoRows: 'max-content',
```

Comment:
`gridAutoRows: max-content` is needed because when you have a larger height than the actual needed, it will stretch and mess up the spacing:
![image](https://user-images.githubusercontent.com/5953191/192845891-eb226045-b2b7-465f-bfbb-0e88ca522433.png)

for the columns it's the same issue, but it's worse since the coin will stretch and the text will be off:
![image](https://user-images.githubusercontent.com/5953191/192846042-fa145071-37db-4f5e-b591-42ee21728cef.png)